config/checkstyle-non-main-files-checks.xml

```diff
@@ -19,7 +19,7 @@
   <module name="RegexpOnFilename">
     <property name="id" value="executablesLocation"/>
     <property name="folderPattern" value="[\\/].ci([\\/]|$)"/>
-    <property name="fileNamePattern" value="\.(bat|cmd|groovy|sh|md)$"/>
+    <property name="fileNamePattern" value="\.(bat|cmd|groovy|sh|md|benchmark)$"/>
```

Comment:
Exclude `benchmark` files since they must are located in `.ci` folder but are not executable.